docs/build_node_setup.md

```diff
@@ -0,0 +1,62 @@
+---
+id: build_node_setup
+title: Set up a development node
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section runs you through the process of setting up a local HydraDX node for development. 
```

Comment:
Is it really just "setting up a local HydraDX node for development"? 
The process creates a complete dev chains where multiple nodes could interact with.
@jak-pan  WDYT?

Reply:
local HydraDX chain instance for development?